src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -41,27 +40,28 @@
 
     private static final Log log = Log.getInstance(TheoreticalSensitivity.class);
     private static final int SAMPLING_MAX = 600; //prevent 'infinite' loops
-    private static final int MAX_CONSIDERED_DEPTH = 1000; //no point in looking any deeper than this, otherwise GC overhead is too high.
+    private static final int MAX_CONSIDERED_DEPTH = 10000; //no point in looking any deeper than this, otherwise GC overhead is too high.
```

Comment:
what if someone wants super low AF sensitivity? will the code still be able to estimate it? should this be a parameter rather than a static final?